src/Codeception/Verify/Verifiers/VerifyBaseObject.php

```diff
@@ -46,4 +46,30 @@ public function notHasAttribute(string $attributeName, string $message = ''): se
         Assert::assertObjectNotHasAttribute($attributeName, $this->actual, $message);
```

Comment:
Yes, in theory, the old methods could call the new ones, or directly the new phpunit assertions.

That might be an idea for a temporary solution until the old ones are dropped completely.

I actually thought to keep them around since this library supports both 9.x and 10.x of phpunit, otherwise they could probably be dropped completely.

But, since they've been added in 9.6.11, it's probably the best solution to keep the old ones, alias them to the new assertions, and raise the minimum 9.x version, as you suggested. That wouldn't require a BC release.

Will update accordingly